app/views/hyrax/embargoes/_list_deactivated_embargoes.html.erb

```diff
@@ -0,0 +1,10 @@
+<% assets_with_deactivated_embargoes.each do |curation_concern| %>
+  <%= link_to curation_concern, edit_embargo_path(curation_concern) %> <%= visibility_badge(curation_concern.visibility) %>
+  <ul>
+    <% curation_concern.embargo_history.each do |entry| %>
+      <% unless entry.empty? %>
```

Comment:
Prefer `if entry.present?` (unless statements can spiral into confusing things to parse; imagine adding another condition to this statement)